### PR TITLE
correct query and graph name

### PR DESCRIPTION
**Pull request by qiliRedHat**
## Type of change

- [ ] Refactor
- [ ] New feature
- [x] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description

1. The graph name 'request duration - 99th quantile' is not correctly match to the actual query.
2. Update ovnkube-master to ovnkube-control-plane

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

**Review by vishnuchalla — APPROVED** [review]
/lgtm.
@mohit-sheth can you confirm that this doesn't break anything please?

File: assets/ovn-monitoring/queries.libsonnet [hunk 1]
```diff
@@ -35,14 +35,14 @@ local generateTimeSeriesQuery(query, legend) = [
             generateTimeSeriesQuery('count(ovn_controller_monitor_all) by (namespace)','')
     },
 
-    ovnKubeMasterCPU: {
+    ovnKubeControlPlaneCPU: {
         query():
-            generateTimeSeriesQuery('irate(container_cpu_usage_seconds_total{pod=~"ovnkube-master.*",namespace="openshift-ovn-kubernetes",container!~"POD|"}[2m])*100','{{container}}-{{pod}}-{{node}}')
+            generateTimeSeriesQuery('irate(container_cpu_usage_seconds_total{pod=~"ovnkube-control-plane.*",namespace="openshift-ovn-kubernetes",container!~"POD|"}[2m])*100','{{container}}-{{pod}}-{{node}}')
```
[the remark this answers is not in the document]
Comment (qiliRedHat) [reply assets/ovn-monitoring/queries.libsonnet]: @rsevilla87 Thanks for the review. I've updated the code.

**Comment by qiliRedHat** [comment]
@rsevilla87 @mohit-sheth Please help to review this again after my change. Thanks.

**Review by venkataanil — APPROVED** [review]
This doesn't brake anything. We can merge it.

**Review by rsevilla87 — APPROVED** [review]
lgtm

**Pull request merged**